### ISSUE TITLE
Carotidynia

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10471]
**Preferred term label:**

**Synonyms**
Carotidodynia (PMID: 28701469)
Carotydinia (PMID: 30338248)
Carotidinia (PMID: 34907753)
carotid artery tenderness (PMID: 37525862)

**Definition (free text, please give PubMed ID)**

PMID: [32809676] Carotidynia, also known as Fay syndrome or TIPIC Syndrome, is a rare vascular disorder that presents with atypical pain in the face and neck area. The disorder is classified as an idiopathic neck pain syndrome, which is associated with point tenderness over the carotid artery bifurcation.

PMID: 38522944 Carotidynia, also known as Fay Syndrome, manifests as an atypical facial neuralgia characterized by an unusual neck pain extending towards the head and associated with carotid artery tenderness. Diagnostic criteria include neck tenderness, elevated carotid pulse without anatomical abnormalities, and neck distension. It was initially classified as a vascular headache but later re-evaluated and reclassified as a nonentity—a general condition caused by nonvascular factors. The etiology has not been extensively elucidated.

**Parent term (use hpo.jax.org/app)**
Pain in head and neck region HP:0046506

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:3287 Takayasu arteritis

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
this seems to be a disease entity rather than a phenotypic feature?

**Comment by pnrobinson** [comment]
@carolina-f 
upon review this is fine

Adding new term: 
- ID: HP:6000944
- label: Carotidynia
- parents: [Pain in head and neck region (HP:0046506)](https://hpo.jax.org/app/browse/term/HP:0046506)
- pmid: PMID:32809676|PMID:38522944
- def: Pain or tenderness in the region of the carotid artery.
- comment: Fay Syndrome is characterized by carotidynia and manifests as an atypical facial neuralgia characterized by an unusual neck pain extending towards the head and associated with carotid artery tenderness. Diagnostic criteria include neck tenderness, elevated carotid pulse without anatomical abnormalities, and neck distension.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10471
- synonyms:
